grails-app/views/contact.gsp

```diff
@@ -0,0 +1,205 @@
+<!doctype html>
+<html lang="en">
+<head>
```

Comment:
A lot of this `<head>` section already exists in the `main.gsp` layout file. as such it isn't really necessary to re-add them